Godeps

```diff
@@ -28,6 +28,7 @@ github.com/hashicorp/consul 5aa90455ce78d4d41578bafc86305e6e6b28d7d2
 github.com/hpcloud/tail b2940955ab8b26e19d43a43c4da0475dd81bdb56
 github.com/influxdata/config b79f6829346b8d6e78ba73544b1e1038f1f1c9da
 github.com/influxdata/influxdb e094138084855d444195b252314dfee9eae34cab
+github.com/influxdata/telegraf 59e981d971ee6b1f97427b7cf2f15783234d59ca
```

Comment:
this is creating problems, why have you run `gdm save` if you're not adding any dependencies?